Samples/Swift/nhubsample/nhubsample/AppDelegate.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  Copyright © 2018 Microsoft All rights reserved.
+//  Licensed under the Apache License (2.0).
```

Comment:
The repo has an MIT license, we shouldn't mix licenses here.
Do we need copyright notices in every file? Especially since the repo has an MIT license already?

Reply:
No, that repo was created with the wrong license. The original code was licensed under Apache 2.0: https://github.com/Azure/azure-notificationhubs/blob/master/LICENSE.txt

We can't change the license. We need to preserve the Apache 2.0 license on the new repos.

Reply:
OK, so put the appropriate license file in that folder of the repo?

Reply:
We need to fix the LICENSE.txt file in the root of the repo. It should be the same contents as the LICENSE.txt file from the old repo: https://github.com/Azure/azure-notificationhubs/blob/master/LICENSE.txt

@iTOYS please fix the LICENSE.txt file in the root.

Reply:
Fixed in separate PR
https://github.com/Azure/azure-notificationhubs-ios/pull/24